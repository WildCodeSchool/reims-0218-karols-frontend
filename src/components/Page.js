import React, { Component } from "react"
import Header from "./Header"
import ReservationTitle from "./ReservationTitle"
import ListChoiceService from "./ListChoiceService"
import ResultCalendar from "./ResultCalendar"
import ListChoiceGender from "./ListChoiceGender"
import { Container, Row, Col } from "reactstrap"
import PrestationLabel from "./PrestationLabel"
import ListChoicePrestation from "./ListChoicePrestation"
import SelectClient from "./SelectClient"
import InfoClient from "./InfoClient"
import Footer from "./Footer"
import ValidationReservationButton from "./ValidationReservationButton"

class Page extends Component {
  render() {
    return (
      <Container>
        <Row>
          <Col className="text-center">
            <Header />
          </Col>
        </Row>
        <hr />
        <Row>
          <Col className="text-center mt-1 mb-3">
            <ReservationTitle />
          </Col>
        </Row>
        <ListChoiceService />
        <hr />
        <Row>
          <Col className="text-center">
            <ListChoiceGender />
          </Col>
        </Row>
        <hr />
        <Row>
          <Col className="text-center">
            <PrestationLabel />
          </Col>
        </Row>
        <ListChoicePrestation />
        <hr />
        <Row>
          <Col className="text-center">
            <SelectClient />
          </Col>
        </Row>
        <Row>
          <Col className="text-center">
            <InfoClient />
          </Col>
        </Row>
        <Row>
          <Col className="text-center">
            <ResultCalendar />
          </Col>
        </Row>
        <Row>
          <Col className="text-center">
            <ValidationReservationButton />
          </Col>
        </Row>
        <Row>
          <Col className="text-center">
            <Footer />
          </Col>
        </Row>
      </Container>
    )
  }
}

export default Page
