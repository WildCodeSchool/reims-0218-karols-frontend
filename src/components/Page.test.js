import React from "react"
import ReactDOM from "react-dom"
import Page from "./Page"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Page />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<Page />).toJSON()
  expect(tree).toMatchSnapshot()
})
