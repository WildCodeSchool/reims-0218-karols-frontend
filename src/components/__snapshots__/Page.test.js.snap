// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <header>
        <img
          alt="logo"
          src="http://via.placeholder.com/100x100"
        />
        <div
          className="dropdown"
          onKeyDown={[Function]}
        >
          <button
            aria-expanded={false}
            aria-haspopup={true}
            className="dropdown-toggle btn btn-secondary"
            onClick={[Function]}
            type="button"
          >
            Change your shop
          </button>
          <div
            aria-hidden={true}
            className="dropdown-menu"
            role="menu"
            tabIndex="-1"
            x-placement={undefined}
          >
            <h6
              className="dropdown-header"
              onClick={[Function]}
              tabIndex="-1"
              type={undefined}
            >
              Header
            </h6>
            <button
              className="disabled dropdown-item"
              disabled={true}
              onClick={[Function]}
              tabIndex="-1"
              type="button"
            >
              Action
            </button>
            <button
              className="dropdown-item"
              onClick={[Function]}
              tabIndex="0"
              type="button"
            >
              Another Action
            </button>
            <div
              className="dropdown-divider"
              onClick={[Function]}
              tabIndex="-1"
              type={undefined}
            />
            <button
              className="dropdown-item"
              onClick={[Function]}
              tabIndex="0"
              type="button"
            >
              Another Action
            </button>
          </div>
        </div>
      </header>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div>
        <h3>
          <span
            className="badge badge-secondary"
          >
            Créez votre réservation
          </span>
        </h3>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-lg-4"
    >
      <div>
        <div
          className="card"
        >
          <div
            className="card-body"
          >
            <h5
              className="card-title"
            >
               
              Préparation
               
            </h5>
          </div>
          <a
            href=""
          >
            <img
              alt="Name of service"
              className="img-fluid"
              height="250"
              src="https://images.unsplash.com/photo-1481501940778-c8bb63e376c5?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=842a2f0d2f40bb7bb59bbdfd78289a0f&auto=format&fit=crop&w=1050&q=80"
              width="250"
            />
          </a>
          <div
            className="card-body"
          >
            <p
              className="card-text"
            >
               
              Description Préparation
               
            </p>
          </div>
          <div
            className="card-footer"
          >
            <button
              className="btn btn-secondary"
              onClick={[Function]}
              type={undefined}
            >
               Choisir ce service 
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-lg-4"
    >
      <div>
        <div
          className="card"
        >
          <div
            className="card-body"
          >
            <h5
              className="card-title"
            >
               
              Table
               
            </h5>
          </div>
          <a
            href=""
          >
            <img
              alt="Name of service"
              className="img-fluid"
              height="250"
              src="https://images.unsplash.com/uploads/1412198485051133af17f/5049dacb?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=dddba89812ed0325b966bf07aba683e3&auto=format&fit=crop&w=1050&q=80"
              width="250"
            />
          </a>
          <div
            className="card-body"
          >
            <p
              className="card-text"
            >
               
              Description Table
               
            </p>
          </div>
          <div
            className="card-footer"
          >
            <button
              className="btn btn-secondary"
              onClick={[Function]}
              type={undefined}
            >
               Choisir ce service 
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-lg-4"
    >
      <div>
        <div
          className="card"
        >
          <div
            className="card-body"
          >
            <h5
              className="card-title"
            >
               
              Table + Préparation
               
            </h5>
          </div>
          <a
            href=""
          >
            <img
              alt="Name of service"
              className="img-fluid"
              height="250"
              src="https://images.unsplash.com/photo-1521001819313-f84e889dd60f?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=4138705d19bea36b2955d3e799285feb&auto=format&fit=crop&w=634&q=80"
              width="250"
            />
          </a>
          <div
            className="card-body"
          >
            <p
              className="card-text"
            >
               
              Description Table & Préparation
               
            </p>
          </div>
          <div
            className="card-footer"
          >
            <button
              className="btn btn-secondary"
              onClick={[Function]}
              type={undefined}
            >
               Choisir ce service 
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div>
        <div
          className="form-check"
        >
          <label
            className="form-check-label "
            htmlFor={undefined}
          >
            <input
              className="form-check-input"
              name="gender"
              type="radio"
            />
             Femme
          </label>
        </div>
        <div
          className="form-check"
        >
          <label
            className="form-check-label "
            htmlFor={undefined}
          >
            <input
              className="form-check-input"
              name="gender"
              type="radio"
            />
             Homme
          </label>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div>
        <p>
          Choix de votre préparation Maquillage
        </p>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="row"
    >
      <div
        className="col-3"
      >
        <div>
          <div
            className="card text-white"
          >
            <img
              alt="Prestations"
              className="img-fluid"
              src="http://via.placeholder.com/250x250"
              width="250"
            />
            <div
              className="card-img-overlay"
            >
              <h5
                className="card-title"
              >
                Card Title
              </h5>
              <p
                className="card-text"
              >
                Choisissez votre prestation
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-3"
      >
        <div>
          <div
            className="card text-white"
          >
            <img
              alt="Prestations"
              className="img-fluid"
              src="http://via.placeholder.com/250x250"
              width="250"
            />
            <div
              className="card-img-overlay"
            >
              <h5
                className="card-title"
              >
                Card Title
              </h5>
              <p
                className="card-text"
              >
                Choisissez votre prestation
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-3"
      >
        <div>
          <div
            className="card text-white"
          >
            <img
              alt="Prestations"
              className="img-fluid"
              src="http://via.placeholder.com/250x250"
              width="250"
            />
            <div
              className="card-img-overlay"
            >
              <h5
                className="card-title"
              >
                Card Title
              </h5>
              <p
                className="card-text"
              >
                Choisissez votre prestation
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-3"
      >
        <div>
          <div
            className="card text-white"
          >
            <img
              alt="Prestations"
              className="img-fluid"
              src="http://via.placeholder.com/250x250"
              width="250"
            />
            <div
              className="card-img-overlay"
            >
              <h5
                className="card-title"
              >
                Card Title
              </h5>
              <p
                className="card-text"
              >
                Choisissez votre prestation
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="select-client"
        >
          Select client : 
        </label>
        <input
          className="form-control"
          id="select-client"
          max="14"
          min="0"
          step="1"
          type="number"
          value="0"
        />
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <form
        className=""
      >
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="exampleEmail"
          >
            Name
          </label>
          <input
            className="form-control"
            id="exampleEmail"
            name="email"
            placeholder="Sandrine"
            type="email"
          />
        </div>
        <div>
          <div
            className="form-check"
          >
            <label
              className="form-check-label "
              htmlFor={undefined}
            >
              <input
                className="form-check-input"
                name="gender"
                type="radio"
              />
               Femme
            </label>
          </div>
          <div
            className="form-check"
          >
            <label
              className="form-check-label "
              htmlFor={undefined}
            >
              <input
                className="form-check-input"
                name="gender"
                type="radio"
              />
               Homme
            </label>
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div
        className="table-responsive"
      >
        <table
          className="table"
        >
          <tbody>
            <tr>
              <td>
                <div>
                  <div
                    className="text-center"
                  >
                    <p>
                      Vendredi
                    </p>
                    <p>
                      25 mai
                    </p>
                  </div>
                  <div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                  </div>
                </div>
              </td>
              <td>
                <div>
                  <div
                    className="text-center"
                  >
                    <p>
                      Vendredi
                    </p>
                    <p>
                      25 mai
                    </p>
                  </div>
                  <div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                  </div>
                </div>
              </td>
              <td>
                <div>
                  <div
                    className="text-center"
                  >
                    <p>
                      Vendredi
                    </p>
                    <p>
                      25 mai
                    </p>
                  </div>
                  <div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                  </div>
                </div>
              </td>
              <td>
                <div>
                  <div
                    className="text-center"
                  >
                    <p>
                      Vendredi
                    </p>
                    <p>
                      25 mai
                    </p>
                  </div>
                  <div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                    <div>
                      <a
                        className="alert-link"
                        href="#"
                      >
                        <p
                          className="text-center"
                        >
                          <span
                            className="badge badge-secondary"
                          >
                            09:00
                          </span>
                        </p>
                      </a>
                    </div>
                  </div>
                </div>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div
        className="form-group"
      >
        <button
          className="btn btn-primary btn-lg active"
          onClick={[Function]}
          type={undefined}
        >
          Récapitulatif
        </button>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="text-center col"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            linkedin
          </div>
          <div
            className="col"
          >
            projet
          </div>
          <div
            className="col"
          >
            liens
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            linkedin
          </div>
          <div
            className="col"
          >
            projet
          </div>
          <div
            className="col"
          >
            liens
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            linkedin
          </div>
          <div
            className="col"
          >
            projet
          </div>
          <div
            className="col"
          >
            liens
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
